myintegrations/gitea-repositories/user1/repo1.git/hooks/post-receive.d/gitea

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+"/home/ethantkoenig/.gopath/src/code.gitea.io/gitea/gitea" hook --config='integrations/app.ini' post-receive
```

Comment:
This will break for everyone... Don't put the test-repos in here, have the test pull it from somewhere (or untar/unzip)